Files/Filesystem/Drives.cs

```diff
@@ -203,7 +131,7 @@ private void DeviceAdded(DeviceWatcher sender, DeviceInformation args)
 
                 drivesList.Add(driveItem);
             }
-            // Update the collection on the ui-thread.
```

Comment:
It is impossible to update a collection like this off the UI thread so ignore these comments change.